docs/usage.md

```diff
@@ -244,16 +368,19 @@ RUN dnf -y install \
     dnf clean all
 ```
 
-Build the image:
+This container build might be what we are familiar with already as we are not using Cachi2 or enforcing network
+isolation.
 
 ```shell
 podman build . -f Containerfile.baseimage --tag atomic-reactor-base-image:latest
 ```
 
-### Build the application image (pip)
+#### Build the application image (pip)
 
 We will base the final application image on our custom base image. The base image build installed all the RPMs we will
-need, so the final phase can use network isolation again 🎉.
+need, so the final phase can use network isolation again 🎉. In order to support the network isolated build, we need
+to remember to `source` the environment file in the step that executes `pip install`. Remember, the source code in
+`/src/atomic-reactor` has also been changed so that the dependencies are pointing to the cached versions.
```

Comment:
Related to the above, maybe call out that this is specifically only osbs-client becomes it comes from github

Reply:
Changed the last line to be 

> Because `osbs-client` comes from
GitHub, the source code in `/src/atomic-reactor` has also been changed so that the dependencies are pointing to the cached
versions.